src/modules/identity/sagas.ts

```diff
@@ -168,77 +165,33 @@ function* handleLogout(_action: LogoutAction) {
   if (address) {
     yield put(disconnectWallet())
     yield put(destroyIdentity(address))
-    // Clear the identity from the SSO iframe. Doing so will log you out of all DCL applications (That use SSO).
-    yield call(clearIdentity, address)
+    yield call(localStorageClearIdentity, address)
   }
 }
 
 function* handleConnectWalletSuccess(action: ConnectWalletSuccessAction) {
   const { wallet } = action.payload
   const { address, providerType } = wallet
 
-  const isAuthDappEnabled: boolean = yield select(getIsAuthDappEnabled)
+  const identity = localStorageGetIdentity(address)
 
-  if (isAuthDappEnabled) {
-    const identity = localStorageGetIdentity(address)
-    if (identity) {
-      yield put(generateIdentitySuccess(address, identity))
-      yield put(loginRequest(providerType, true))
-    } else {
-      redirectToAuthDapp()
-    }
-    return
-  }
-
-  // Obtains the identity from the SSO iframe.
-  const identity: AuthIdentity | null = yield call(getIdentity, address)
-
-  // If an identity is found, store it and proceed with the login so the state acknowledges that the user is connected.
-  // If not, proceed with the login in order to retrieve user's signature.
   if (identity) {
     yield put(generateIdentitySuccess(address, identity))
     yield put(loginRequest(providerType, true))
   } else {
-    yield put(loginRequest(providerType))
+    redirectToAuthDapp()
   }
 }
 
 function* handleChangeAccount(action: ChangeAccountAction) {
   const { wallet } = action.payload
   const { address, providerType } = wallet
 
-  const isAuthDappEnabled: boolean = yield select(getIsAuthDappEnabled)
-
-  if (isAuthDappEnabled) {
-    const identity = localStorageGetIdentity(address)
-    if (identity) {
-      yield put(generateIdentitySuccess(address, identity))
-      yield put(loginRequest(providerType, true))
-    } else {
-      redirectToAuthDapp()
-    }
-    return
-  }
-
-  // Obtains the identity from the SSO iframe.
-  const identity: AuthIdentity | null = yield call(getIdentity, address)
-
-  // If an identity is found, store it and proceed with the login so the state acknowledges that the user is connected.
-  // If not, proceed with the login in order to retrieve user's signature.
+  const identity = localStorageGetIdentity(address)
   if (identity) {
     yield put(generateIdentitySuccess(address, identity))
     yield put(loginRequest(providerType, true))
   } else {
-    yield put(loginRequest(providerType))
-  }
-
-  yield put(clearAssetPacks())
```

Comment:
should we keep this `clearAssetPacks`?

Reply:
nice catch 😱

Reply:
The auth dapp flow returned before doing so, but it might have been a bug from before

Reply:
you're right! with the new auth dapp flow, this is no longer required because when the user is redirected again to the builder, the state is cleared

Reply:
But the user might change account via the wallet, which event will end up calling the saga. If the changed wallet already has an identity, it ahould clear the assetpacks and whatever it did before.

Reply:
good catch! this is the way 👍